packages/map/lib/src/leaflet/services/map.service.ts

```diff
@@ -16,7 +17,7 @@ export class MapService {
 		if (isPlatformBrowser(this.platformId)) {
 			this.L = require('leaflet');
 			this.esri = require('esri-leaflet');
-			require('leaflet-draw');
+			this.geoman = require('@geoman-io/leaflet-geoman-free');
```

Comment:
Please don't use the require statement anymore. Switch to `import * as A from B`. Also, inject the dependencies via Angular dependency injection instead of requesting them in a constructor.